src/TestRunnerOrchestrator.ts

```diff
@@ -168,7 +168,7 @@ export default class TestRunnerOrchestrator {
   }
 
   private createTestRunnerSandboxes(): Promise<TestRunnerSandbox[]> {
-    let cpuCount = os.cpus().length;
+    let cpuCount = this.options.cpuCount || os.cpus().length;
     let testRunnerSandboxes: TestRunnerSandbox[] = [];
     let allPromises: Promise<any>[] = [];
     log.info(`Creating ${cpuCount} test runners (based on cpu count)`);
```

Comment:
We may want to change this logging statement as it's not always based on cpu count. We can either have two logging statements *(based on cpu count)* and *(based on config)* or just have one logging statement and not mention where we get the amount from. What do you think?

CC @nicojs